### ISSUE TITLE
Feature request: reply

### ISSUE THREAD

**Issue by kseistrup** [traviskaufman/YATI#3]
Please implement

``` sh
-i ID, --in-reply-to ID    send a tweet in reply to tweet #ID
```

so that conversations are kept together.

**Comment by traviskaufman** [comment]
[done](https://github.com/traviskaufman/YATI/commit/93fa6a099ab5753ebba77444283ad7a88a28e1d3). Will be included in 1.1.0 release (which will be out sooner than later, I've been so damn busy lately X_X). Sorry for such a long wait!

**Comment by kseistrup** [comment]
Cool. :)
